app/controllers/organizations_controller.rb

```diff
@@ -1,4 +1,15 @@
 # frozen_string_literal: true
 
 class OrganizationsController < ApplicationController
+  before_action :set_organization, only: %i[public]
+
+  def public
+    render
+  end
```

Comment:
Podrías probar que tal vez el método render no sea necesario acá:
```suggestion
  def public; end
```

Reply:
cambio realizado, gracias